tokio-console/src/view/help.rs

```diff
@@ -0,0 +1,70 @@
+use tui::{
+    layout::{self, Constraint, Direction, Layout},
+    text::Text,
+    widgets::{Clear, Paragraph, Wrap},
+};
+
+use crate::{state::State, view};
+
+/// Simple view for help popup
+pub(crate) struct HelpView<T> {
+    help_text: Option<T>,
+}
+
+impl<T> HelpView<T>
+where
+    T: Into<Text<'static>>,
+{
+    pub(super) fn new(help_text: T) -> Self {
+        HelpView {
+            help_text: Some(help_text),
+        }
+    }
+
+    pub(crate) fn render<B: tui::backend::Backend>(
+        &mut self,
+        styles: &view::Styles,
+        frame: &mut tui::terminal::Frame<B>,
+        _area: layout::Rect,
+        _state: &mut State,
```

Comment:
is there a reason this has these arguments, when it doesn't use them? since this isn't a trait method, I think we should just not take these arguments just because they are passed to other views' `render` methods.
```suggestion
```